README.md

```diff
@@ -1,101 +1,77 @@
+# Welcome to the Viam Documentation
 
-## Dependencies
+> **Note**
+> Looking to contribute? Check out the [Contributer Guide](./CONTRIBUTING.md).
 
-* npm
-  * Install 16 (LTS) from [https://nodejs.org/en/download/](https://nodejs.org/en/download/).
+## Build the docs locally
 
-* Hugo
-  * macOS/Linux: `brew install hugo`
-  * Windows: [https://gohugo.io/getting-started/installing/](https://nodejs.org/en/download/)
+To be able to build the docs locally, you need to install the following:
 
-## Local Development
+- [`npm`](https://nodejs.org/en/download/).
+- Hugo
+  - macOS/Linux: `brew install hugo`
+  - Windows: [https://gohugo.io/getting-started/installing/](https://nodejs.org/en/download/)
 
-* Production views
-  * Standard production: `make serve-prod`
-  * Production with draft files: `make serve-prod-draft`
-  * Production with future files: `make serve-prod-future`
-* Development views (likely not needed much)
-  * Standard development: `make serve-dev`
-  * Development with draft files: `make serve-dev-draft`
-  * Development with future files: `make serve-dev-future`
+You can build the docs for local development using the following command:
 
-* You can just run `hugo serve` after installing Hugo to show the prod view.
-
-## Building
+```sh
+make serve-dev
+```
 
-This will build and serve `/public` from a separate, non-Hugo server, at [http://localhost:9000](http://localhost:9000).
+If you would like to see the production view (without drafts and with minified CSS), run:
 
-`make build-prod && python3 -m http.server 9000 --directory public`
+```sh
+make serve-prod
+```
 
-## Vale Linting
+You can also run `hugo serve` after installing Hugo to show the production view.
 
-When you open a PR, your changes will be checked against a few style rules.
-To run this check locally, follow the instructions in the [Vale Readme](.github/vale/README.md).
-
-## Publishing
+### Generate HTML docs
 
-This is handled automatically by a GitHub Action on push.
+To generate the full HTML version of the docs run:
 
-## Contributing
+```sh
+make build-prod
+```
 
-Don't hesitate to create a pull request.
-Every contribution is appreciated.
+You can serve the resulting docs with:
 
-### Notes for contributors
+```sh
+python3 -m http.server 9000 --directory public
+```
 
-* Don't make changes on the main repo, do them in your fork.
-* Every sentence should be on a new line.
-* Test locally before submitting a PR.
-* Always submit a PR before merging.
-* The author of the PR should merge unless they cannot and it's time-sensitive.
+## Publishing
 
-1. Fork the official repo into your personal GitHub.
-2. The new forked copy is cloned to your local system.
-3. Git remote path for the 'official' repository is added to the local clone: ```git remote add upstream git@github.com:viamrobotics/docs.git```
+The docs are automatically published when a PR merges.
 
-When you are ready to contribute changes to the docs:
+## Style Guide
```

Comment:
Should this section instead be part of CONTRIBUTING?
Or, what is the dividing line between this readme and contributing/could it all just be in readme?

Reply:
I'll move it